daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/EvaluationOrderTest.scala

```diff
@@ -2149,6 +2165,27 @@ class EvaluationOrderTest extends AnyFreeSpec with Matchers with Inside {
         }
       }
 
+      "a disclosed contract" - {
+
+        // TEST_EVIDENCE: Semantics: Evaluation order of fetch of a wrongly typed disclosed contract
+        "wrongly typed contract" in {
```

Comment:
Similar testing for `fetch_by_key` and `lookup_by_key` are not possible as we have no analogous `COERCE_CONTRACT_KEY` command.